apps/merkle_patricia_tree/lib/db/rocksdb.ex

```diff
@@ -0,0 +1,41 @@
+defmodule MerklePatriciaTree.DB.RocksDB do
+  @moduledoc """
+  Implementation of MerklePatriciaTree.DB which
+  is backed by rocksdb.
+  """
+
+  alias MerklePatriciaTree.{Trie, DB}
+
+  @behaviour MerklePatriciaTree.DB
+
+  @doc """
+  Performs initialization for this db.
+  """
+  @spec init(DB.db_name()) :: DB.db()
+  def init(db_name) do
+    {:ok, db_ref} = Rox.open(db_name, create_if_missing: true)
+
+    {__MODULE__, db_ref}
+  end
+
+  @doc """
+  Retrieves a key from the database.
+  """
+  @spec get(DB.db_ref(), Trie.key()) :: {:ok, DB.value()} | :not_found
+  def get(db_ref, key) do
+    case Rox.get(db_ref, key) do
```

Comment:
What's this case do? Can we just return `Rox.get(db_ref, key)` directly?

Reply:
good spot, it is a leftover from the leveldb implementation